app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -68,8 +68,43 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class EndExperiment(graphene.Mutation):
+    nimbus_experiment = graphene.Field(NimbusExperimentType)
+    message = ObjectField()
+    status = graphene.Int()
+
+    class Arguments:
+        input = ExperimentIdInput(required=True)
+
+    @classmethod
+    def mutate(cls, root, info, input: ExperimentIdInput):
+        experiment = NimbusExperiment.objects.get(id=input.id)
+
+        if experiment.status == NimbusExperiment.Status.LIVE:
+            experiment.is_end_requested = True
+            experiment.save()
+            obj = experiment
+            msg = "success"
+        else:
+            obj = None
+            status = experiment.status
+            msg = (
+                f"Nimbus Experiment has status '{status}', but can only "
+                "be ended when set to 'Live'."
+            )
```

Comment:
Also added a little bit of status validation here because I was able to set a Draft status experiment to end and it proceeded from Live to Completed after I pushed it. Probably not the end of the world, but not useful